### PR TITLE
Add a test for installing a plugin

### PR DESCRIPTION
**Pull request by hhorak**
This will fail until the plugin dir is fixed in the base.cnf file (fix suggested in #235).

### CODE REVIEW

**Comment by phracek** [comment]
Test failed:

```
-----------------------------------------------
Running test run_plugin_test (starting at 2024-04-05 14:32:09+00:00) ... 
-----------------------------------------------
  Testing plugin installation
    Created container 66f8fd9a9e55803a746273b23524e4da95ce81229c1a1199f67ba5f9fa9b839d
  Testing MySQL connection to 10.88.0.174...
    Trying to connect....... OK
  Success!
ERROR 1126 (HY000) at line 1: Can't open shared library '/usr/lib64/mysql/plugin/sql_errlog.so' (errno: 2, cannot open shared object file: No such file or directory)
Test for image 'quay.io/sclorg/mariadb-103-c8s:1' FAILED (exit code: 1)
ERROR 1146 (42S02) at line 1: Table 'mysql.IdonotExist' doesn't exist
grep: /var/lib/mysql/data/sql_errors.log: No such file or directory
Test for image 'quay.io/sclorg/mariadb-103-c8s:1' FAILED (exit code: 2)
Test for image 'quay.io/sclorg/mariadb-103-c8s:1' FAILED (exit code: 2)
Tests finished with EXIT=0

==============================================
```

But it looks like it needs #235

**Comment by hhorak** [comment]
[test]

File: test/run [hunk 1]
```diff
@@ -691,5 +692,30 @@ function run_upgrade_test() {
...
+    "${test_name}" \
+    --env MYSQL_ROOT_PASSWORD=rootpass \
+    --env MYSQL_USER=user \
+    --env MYSQL_PASSWORD=foo \
+    --env MYSQL_DATABASE=db
+
+  test_connection "${test_name}" root rootpass
+  ip=$(ct_get_cip "${test_name}")
+
+  mysql_cmd "$ip" "root" "rootpass" -e "INSTALL PLUGIN SQL_ERROR_LOG SONAME 'sql_errlog' \G"
```
Comment (phracek) [inline test/run:710]: SQL_ERROR_LOG is the plugin or SONAME? I do not follow.
It is not clear to me as a database dummy folk. Is it documented somewhere?

If not, can you please add this to documentation?

**Review by phracek — CHANGES_REQUESTED** [review]
The tests passed, but I have missed the documentation about plugin.

**Comment by hhorak** [comment]
This test only follows what is documented in upstream: https://mariadb.com/kb/en/sql-error-log-plugin/
I don't see a need to duplicate upstream documentation here.

**Pull request merged**